packages/slice-machine/lib/models/common/widgets/StructuredText/Mock/Form.js

```diff
@@ -23,10 +25,12 @@ const HandlePatternTypes = ({
 }) => {
   const { theme } = useThemeUI();
 
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
```

Comment:
Normally this is not something that we do.
Even though I totally understand that it's your role to fixe those issues, it's still painful to see new errors being ignored.